### ISSUE TITLE
[TODO] update bolt11-decode.js

### ISSUE THREAD

**Issue by dni** [lnbits/lnbits#1885]
//TODO - A reader MUST check that the signature is valid (see the n tagged field)
//TODO - Tagged part of type f: the fallback on-chain address should be decoded into an address format
//TODO - A reader MUST check that the SHA-2 256 in the h field exactly matches the hashed description.
//TODO - A reader MUST use the n field to validate the signature instead of performing signature recovery if a valid n field is provided.

lnbits/static/js/bolt11-decoder.js

**Comment by motorina0** [comment]
@dni is this topic completed?

**Comment by dni** [comment]
nope, its still there. https://github.com/lnbits/lnbits/blob/dev/lnbits/static/js/bolt11-decoder.js

should we maybe switch to bitcoinjs bolt11 library?
